config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:main
```

Comment:
I this change might be unintentional. Can you please drop this?

Reply:
yeah, sorry. Will revert that